src/navigate-pop/tiling-stacking-windows.md

```diff
@@ -0,0 +1,107 @@
+# Manipulating Windows with Tiling & Stacking
+
+This section describes the Auto Tiling, Stacking, and window adjustment features in Pop!\_OS.
+
+---
+## Window Adjustment Mode
+
+Adjustment mode allows users to easily manipulate the placement of windows using keyboard shortcuts.
+
+Place a window into adjustment mode by bringing the window into focus, and then pressing `SUPER` + `Enter`. The window will be highlighted in yellow. You can now manipulate the window with the `🠐` `🠑` `🠒` `🠓` keys in combination with other keys.
+
+<video autoplay loop>
+    <source src="/images/tiling-stacking-windows/adjustment-mode-manipulation.webm" />
+</video>
+
+All keyboard shortcuts available in adjustment mode can be viewed and modified in Settings ➞ Keyboard ➞ View and Customize Shortcuts ➞ Move, resize, and swap windows. You can also view all tiling shortcuts by clicking the tiling button in the upper right corner, and then clicking `View All`.
```

Comment:
`upper right corner of the screen` might be more clear, you were just talking about Settings but we're not talking about the upper right corner of Settings.